test/utils/urls_test.js

```diff
@@ -0,0 +1,18 @@
+const rewire = require('rewire')
+const Urls = rewire('../../utils/urls')
```

Comment:
Similar to comment up at the top, I would reserve this style of Caps casing to strictly `Class` based objects. Since you'll never instantiate this with `new`, and the capital U in `Urls` seems to imply a class, its somewhat indirect.